rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/fragment/CovidPatientRegisterFragmentTest.java

```diff
@@ -0,0 +1,69 @@
+package io.ona.rdt.robolectric.fragment;
+
+import android.content.ContextWrapper;
+import android.content.Intent;
+
+import androidx.fragment.app.testing.FragmentScenario;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.powermock.reflect.Whitebox;
+import org.robolectric.Robolectric;
+import org.robolectric.Shadows;
+
+import io.ona.rdt.R;
+import io.ona.rdt.activity.CovidPatientProfileActivity;
+import io.ona.rdt.activity.CovidPatientRegisterActivity;
+import io.ona.rdt.domain.Patient;
+import io.ona.rdt.fragment.CovidPatientRegisterFragment;
+import io.ona.rdt.presenter.CovidPatientRegisterFragmentPresenter;
+import io.ona.rdt.presenter.PatientRegisterFragmentPresenter;
+import io.ona.rdt.util.CovidConstants;
+import io.ona.rdt.viewholder.CovidPatientRegisterViewHolder;
+import io.ona.rdt.viewholder.PatientRegisterViewHolder;
+
+public class CovidPatientRegisterFragmentTest extends FragmentRobolectricTest {
+
+    private CovidPatientRegisterFragment covidPatientRegisterFragment;
+    private CovidPatientRegisterActivity patientRegisterActivity;
+    private FragmentScenario<CovidPatientRegisterFragment> fragmentScenario;
+
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+
+        patientRegisterActivity = Robolectric.buildActivity(CovidPatientRegisterActivity.class).create().resume().get();
```

Comment:
Why is launching the activity necessary?

Reply:
to test this method https://github.com/onaio/rdt-standard/blob/011f4da9ed68ba8c302e476255177b8ce8cce533/rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/fragment/CovidPatientRegisterFragmentTest.java#L58

Reply:
fixed.